index.html

```diff
@@ -3,77 +3,686 @@
+    
+                <p>Gravity waves are the latest and deepest secret yet pried out of the cosmic
+                microwaves, which were discovered accidentally by Arno Penzias and Robert
+                Wilson at Bell Labs 50 years ago. They won the Nobel Prize.</p>
+    
+                <p>Dr. Kovac has spent his career trying to read the secrets of these waves. 
+                He is one of four leaders of Bicep, which has operated a series of increasingly 
+                sensitive radio telescopes at the South Pole, where the thin, dry air creates 
+                ideal observing conditions. The others are Clement Pryke of the University 
+                of Minnesota, Jamie Bock of the California Institute of Technology and
+                Dr. Kuo of Stanford.</p>
+    
+                <p>“The South Pole is the closest you can get to space and still be on
+                the ground,” Dr. Kovac said. He has been there 23 times, he said, wintering over
+                in 1994. “I’ve been hooked ever since,” he said.</p>
 
+                <figure>
+                    <img class="nytimages1" src="images/nytimages1.jpg" alt="The bicep telescope">
+                    <figcaption class="iframe-span">The Bicep2 telescope, in the foreground, was used to
+                        detect the faint spiraling gravity patterns — the signature of a universe being
+                        wrenched violently apart at its birth.</figcaption>
+                </figure>
+                <p class="caption">Steffen Richter/Associated Press</p>
+    
+                <p>In 2002, he was part of a team that discovered that the microwave radiation was
+                polarized, meaning the light waves had a slight preference to vibrate in one
+                direction rather than another.</p>
+    
+                <p>This was a step toward the ultimate goal of detecting the gravitational waves
+                from inflation. Such waves, squeezing space in one direction and stretching it
+                in another as they go by, would twist the direction of polarization of the 
+                microwaves, theorists said. As a result, maps of the polarization in the sky 
+                should have little arrows going in spirals.</p>
+    
+                <p>Detecting those spirals required measuring infinitesimally small 
+                differences in the temperature of the microwaves. The group’s telescope, Bicep2,
+                is basically a giant superconducting thermometer.</p>
+    
+                <p>“We had no expectations what we would see,” Dr. Kovac said.</p>
+    
+                <p>The strength of the signal surprised the researchers, and they spent a year
+                burning up time on a Harvard supercomputer, making sure they had things right
+                and worrying that competitors might beat them to the breakthrough.</p>
+    
+                <h1>A Special Time</h1>
```

Comment:
#### `REQUIRED`

Consider using `h2-h6` element here instead. `<h1>` should be used only as a top-level heading. 

> Using only h1 elements in a HTML document results in a flat document outline as heading element semantics are conveyed to users as per the numeric in the heading element tag name.

Learn more [here](https://www.w3.org/wiki/HTML/Usage/Headings/h1only)